tests/unit/test-cmd/test.docs.js

```diff
@@ -0,0 +1,23 @@
+/* @flow */
+import {it, describe} from 'mocha';
+import sinon from 'sinon';
+import {assert} from 'chai';
+
+import docs, {openDocs} from '../../../src/cmd/docs';
+
+
+describe('docs', () => {
+
+  function setUp() {
+    const docsResult = sinon.stub(openDocs, 'openURL');
+    return docsResult;
+  }
+
+  it('passes the correct URL to docs', () => {
+    const docsResult = setUp();
+    const expectedURL = 'https://developer.mozilla.org/en-US/Add-ons' +
```

Comment:
nitpicky comment: for consistency with the rest of web-ext, make the camel case for this look like `expectedUrl`.